src/SDKs/HybridDataManager/Management.HybridData/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "hybriddatamanager/resource-manager" -AutoRestVersion "latest" -SpecsRepoFork "AzureHybridDataManager"
```

Comment:
Is the Rest spec PR merged already?

Reply:
No, we are waiting for a linter suppression approval for that PR